Header header
uint16 box_x
uint16 box_y
uint16 box_w
uint16 box_h